pkgs/development/python-modules/aws-sam-translator/default.nix

```diff
@@ -30,6 +30,5 @@ buildPythonPackage rec {
     homepage = "https://github.com/awslabs/serverless-application-model";
     description = "Python library to transform SAM templates into AWS CloudFormation templates";
     license = lib.licenses.asl20;
-    maintainers = [ lib.maintainers.andreabedini ];
```

Comment:
In the future please add an empty maintainer field to make it clear that this package is unmaintained.